### PR TITLE
Fix Sentry issues, update certs.pem

### PR DESCRIPTION
**Pull request by Wawrzyn321**
**Description**

Check out what was fixed in https://sentry.io/organizations/kyma-project/issues/<number>

Changes proposed in this pull request:

#3122020210 was really fun to reproduce...
```
kind: HorizontalPodAutoscaler
apiVersion: autoscaling/v2beta2
metadata:
  name: repro
  namespace: default
spec:
  scaleTargetRef:
    kind: Deployment
    name: necessary-occasion-m9fqn
    apiVersion: apps/v1
  minReplicas: 1
  maxReplicas: 5
  metrics:
    - type: Resource
      resource:
        name: cpu
        target:
          type: Utilization
          averageUtilization: 50
    - type: Object
      object:
        describedObject:
          kind: Deployment
          name: ziom
        target:
          type: "AverageValue"
          averageValue: 2
        metric:
          name: http_requests
          selector: {matchLabels: {verb: GET}}
```

**Related issue(s)**

### CODE REVIEW

File: core-ui/src/shared/components/StatsGraph/StatsGraph.js [hunk 1]
```diff
@@ -31,7 +31,8 @@ function getGeometry(ctx, { scale, hScale, dataPoints }) {
   const width = ctx?.canvas.width - geo.scaleWidth;
   const height = ctx?.canvas.height - geo.scaleHeight;
 
-  geo.sectionWidth = (width - 2 * geo.graphPaddingH) / dataPoints;
+  // make sure geo.sectionWidth is not negative on really tiny screens
+  geo.sectionWidth = Math.abs((width - 2 * geo.graphPaddingH) / dataPoints);
```
Comment (akucharska) [inline core-ui/src/shared/components/StatsGraph/StatsGraph.js:35]: After loading the overview view I see an empty graph. It starts working when I change the timeline from 1h to 3h
Screen Shot 2022-03-24 at 10 42 55

**Review by akucharska — CHANGES_REQUESTED** [review]

File: core-ui/src/shared/components/StatsGraph/StatsGraph.js (hunk 1 above)
Comment (Wawrzyn321) [reply core-ui/src/shared/components/StatsGraph/StatsGraph.js:35]: Yeah, that was here before, it's not connected to Sentry issues.

Comment (akucharska) [reply core-ui/src/shared/components/StatsGraph/StatsGraph.js:35]: Then how it is working on dev?

**Review by akucharska — APPROVED** [review]

**Review by akucharska — APPROVED** [review]

**Pull request merged**